src/webview/static/README.md

```diff
@@ -0,0 +1,15 @@
+# Webview static files
+
+This directory holds static files to be packaged for use by the embedded
+WebView. Files herein (other than those named `README.md`) will be copied out to
+the packaged bundle at build time, into a subdirectory named `webview`.
+
+_N.B._: When editing and testing, changes made to this directory will **not**
+be uploaded to an emulator after a mere reload! You must perform a full rebuild
+(`react-native run-${TARGET}`) to repackage any new or altered assets.
```

Comment:
As I remember it, not terribly long — on the order of 15s, as opposed to 5s. There's no actual recompilation of other components being done, just a very quick scan of the timestamps of their penultimate dependencies (which will normally all be up-to-date). Memory probably is not the best thing to go on for that, though; I'll double-check.

It may be a moot point, though: even for dynamic elements, I find that I've been doing most of my development of this sort by fiddling around in the WebView inspector, and only "freezing" it into code or static assets when I'm nearly done. ¯\_(ツ)_/¯

Reply:
> even for dynamic elements, I find that I've been doing most of my development of this sort by fiddling around in the WebView inspector, and only "freezing" it into code or static assets when I'm nearly done. ¯\_(ツ)_/¯

True enough! I generally do the same thing. That's a much faster cycle still, which is great.

I think in my experience when I'm making webview changes there's typically at least a couple of cycles of actually putting the changes into the tree and wanting to refresh from that. It could be just one if I consistently did that translation correctly (or zero if it were so consistent I didn't feel a need to check), but it's fiddly enough that I think most often I don't. So if doing that took like a minute, or several minutes, that'd be bad.

15s seems OK, though. Thanks for checking :slightly_smiling_face: 

Maybe s/full //? To me "full rebuild" suggests specifically a build that's *not* using any cached intermediate build products.